onnxruntime/core/graph/contrib_ops/nhwc_schema_defs.cc

```diff
@@ -45,6 +45,27 @@ ONNX_MS_OPERATOR_SET_SCHEMA(NhwcMaxPool, 1,
                                   ::onnxruntime::contrib::convPoolShapeInferenceNhwc(ctx, true, true, 0, 1);
                                 }));
 
+ONNX_MS_OPERATOR_SET_SCHEMA(NhwcAvgPool, 1,
```

Comment:
>NhwcAvgPool [](http://example.com/codeflow?start=28&length=11)

what is the reason to not leverage the internal NHWC ops

Reply:
Can't get it to work :-( would you provide some guidance?